index.html

```diff
@@ -21,7 +19,7 @@
 
         <a class="nav-link" href="#pricing">Pricing</a>
       </nav>
-    </header>
+    </div>
```

Comment:
Looks like this section is the header of your app. Due to semantic reasons, I suggest you use `header` tag instead of `div` one